skeleton/etc/zope.conf

```diff
@@ -1,6 +1,9 @@
 %define INSTANCE /app
 instancehome $INSTANCE
 
+%define CLIENTHOME $(CLIENT_HOME)
+clienthome $CLIENTHOME
```

Comment:
Was CLIENT_HOME from docker-entrypoint.sh just ignored before?

Is it possible to do `clienthome $(CLIENT_HOME)` without defining CLIENTHOME?

I think we should only merge this for 6.0.x if we make the same change for 6.1.x.

Reply:
> Was CLIENT_HOME from docker-entrypoint.sh just ignored before?
> 

it was used in zeo.conf (when ZEO configuration), but not in zope.conf, that's why I bring it also here, it will do no harm having it here too.

> Is it possible to do `clienthome $(CLIENT_HOME)` without defining CLIENTHOME?
> 

That's why in `docker-entrypoint.sh` I set its value to be the one defined in the env var (if it exists). The only change is that now it is allowed to set its value through an environment var, beforehand it was hardcoded.

> I think we should only merge this for 6.0.x if we make the same change for 6.1.x.

Yes, I just did the PR for 6.0.x but I will do it also for 6.1.x

Reply:
I see, it makes sense to keep it consistent with zeo.conf and relstorage.conf.